### PR TITLE
models : add userCollection models

### PR DESCRIPTION
**Pull request by prabalsingh24**
User Collection Models

### CODE REVIEW

**Comment by prabalsingh24** [comment]
@MonkeyDo   How should I write tests? Like this [EditorModel-tests](https://github.com/bookbrainz/bookbrainz-data-js/blob/master/test/testEditor.js) or by writing rawSql queries and comparing the result with bookshelf models?

**Comment by MonkeyDo** [comment]
You're going to want tests to ensure the models are working fine. There's two ways of doing it that can be combined: you can test the models themselves (like in the link you sent) or you can manipulate the database with raw queries and see if the model interactions change is as expected.

For example, you could use the ORM model to save a new collection and use the model to fetch the newly created items, or you could use raw sql to check if the new items are in the database as you expect. 
The other side, you would create new items with a raw sql query and fetch them with the ORM model to see if you get what you expect.

That being said, considering we use the ORM models as an interface pretty much all the time (except for a couple of fetch queries here or there), I don't think there's a need for these extra raw sql steps; testing the models should be enough.

File: src/models/userCollection.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+
+import {camelToSnake, snakeToCamel} from '../util';
+
+
+export default function identifier(bookshelf) {
```
Comment (MonkeyDo) [inline src/models/userCollection.js:22]: Here and for the other models, the name hasn't been changed

Comment (prabalsingh24) [reply src/models/userCollection.js:22]: Oops!

**Comment by prabalsingh24** [comment]
> Shall I merge this, or do you want to to see if you have any changes to make?

Yes you can merge this. I will write tests in a different PR

**Pull request merged**